### PR TITLE
Allow filtering on ready images

### PR DESCRIPTION
**Pull request by ChaitanyaKulkarni28**

### CODE REVIEW

File: container_images/gce-img-resource/check/check_command.go [hunk 1]
```diff
@@ -39,10 +39,19 @@ func Run(request Request) (Response, error) {
 	}
 
 	call := computeService.Images.List(request.Source.Project)
+
+	var filter string
+
+	if request.Source.ReadyOnly {
```
Comment (dorileo) [inline container_images/gce-img-resource/check/check_command.go:45]: Can you update the comment starting on line 13?

File: container_images/gce-img-resource/check/check_command.go [hunk 2]
```diff
@@ -39,10 +39,19 @@ func Run(request Request) (Response, error) {
...
+		filter = "(status = READY) "
+	}
+
 	if request.Source.Family != "" {
-		call = call.Filter(fmt.Sprintf("family = %s", request.Source.Family))
+		filter = filter + fmt.Sprintf("(family = %s)", request.Source.Family)
```
Comment (dorileo) [inline container_images/gce-img-resource/check/check_command.go:50]: Did you check if this format is correct/valid? `(status = READY)(family = family-name)` if I were to guess I'd have guessed `(status = READY, family = family-name)`.

Comment (dorileo) [reply container_images/gce-img-resource/check/check_command.go:50]: Nitpick: if the space between the tokens are relevant you could use a `strings.Join([]string{...}, " ")` instead.

Comment (ChaitanyaKulkarni28) [reply container_images/gce-img-resource/check/check_command.go:50]: yep, that's the right format. Reference: https://cloud.google.com/compute/docs/reference/rest/v1/images/list

File: container_images/gce-img-resource/check/check_command.go (hunk 1 above)
Comment (ChaitanyaKulkarni28) [reply container_images/gce-img-resource/check/check_command.go:47]: Done

File: container_images/gce-img-resource/check/check_command.go (hunk 2 above)
Comment (ChaitanyaKulkarni28) [reply container_images/gce-img-resource/check/check_command.go]: > Nitpick: if the space between the tokens is relevant you could use a `strings.Join([]string{...}, " ")` instead.

Done

**Pull request merged**